github_pr.py

```diff
@@ -75,6 +76,78 @@ def _load_issue(**args):
     return repo.get_issue(args['number'])
 
 
+def _validate_list_of_dict(list_of_dict):
+    """
+    Validates that obj is a list containing dictionaries with entries for 'pr' and 'issue'
+    """
+    if list_of_dict:
```

Comment:
are you saying that i just need 
the return, because it will return false if any of those are not true? That makes sense.